### ISSUE TITLE
Weird default suggestion list

### ISSUE THREAD

**Issue by lczech** [yongkangchen/atom-ctags#106]
Original issue: https://github.com/atom/autocomplete-plus/issues/502

In every file (even plain text), autocomplete suggests a list of items that have no use for me when I type a dot:
![suggestions](https://cloud.githubusercontent.com/assets/4741887/8149458/2e10789a-12c4-11e5-825f-d5d836b8d34c.png)

They appear only when atom-ctags is enabled, thus I suspect they are introduced to the autocomplete suggestion list by this package.

**Comment by jcostello** [comment]
Having the same problem

**Comment by lalomartins** [comment]
+1
looking at the tags file, this data definitely comes from there. Must be something in the way atom-ctags is parsing it?

**Comment by baelter** [comment]
Same here. Would also be nice if the provider set correct types for suggestions so we get those fancy icons in the suggestions list.

**Comment by yongkangchen** [comment]
Some one who can tell me how to reproduce this issue? I type dot in css file and not reproduce this issue? Did I missing something?

**Comment by AWTrost** [comment]
@yongkangchen I think it may have something to do with what version/type/binary of ctags that is installed or something, as I'm having the same problem and in the atom dev tools console this is the error: `repetition-operator operand invalid` (i've pasted the full error below).  I used homebrew to install ctags, not sure if the version they use isn't the same? 

```console
atom-ctags: command error, ctags-darwin: Warning: regcomp with[ 	]+([^;]+)[ 	]*?;: repetition-operator operand invalid
ctags-darwin: Warning: regcomp extends[ 	]+['"]([^'"]+)['"][ 	]*?;: repetition-operator operand invalid
ctags-darwin: Warning: regcomp use[ 	]+base[ 	]+['"]([^'"]+)['"][ 	]*?;: repetition-operator operand invalid
ctags-darwin: Warning: regcomp use[ 	]+parent[ 	]+['"]([^'"]+)['"][ 	]*?;: repetition-operator operand invalid
ctags-darwin: Warning: regcomp Mojo::Base[ 	]+['"]([^'"]+)['"][ 	]*?;: repetition-operator operand invalid
ctags-darwin: Warning: regcomp ^[ 	]*?use[ 	]+([^;]+)[ 	]*?;: repetition-operator operand invalid
ctags-darwin: Warning: regcomp ^[ 	]*?require[ 	]+((\w|\:)+): repetition-operator operand invalid
ctags-darwin: Warning: regcomp ^[ 	]*?has[ 	]+['"]?(\w+)['"]?: repetition-operator operand invalid
ctags-darwin: Warning: regcomp ^[ 	]*?\*(\w+)[ 	]*?=: repetition-operator operand invalid
ctags-darwin: Warning: regcomp ^[ 	]*?our[ 	]*?[\$@%](\w+): repetition-operator operand invalid
ctags-darwin: Warning: regcomp ^[ 	]*+([A-Za-z0-9_]+)(;|\([^)]*:[^)]*\)): repetition-operator operand invalid
 /Users/trosta/Work/code.espn.com/sRPMBut/.tags/Users/trosta/.atom/packages/atom-ctags/lib/tag-generator.coffee:81 stderr
```

**Comment by quickgold192** [comment]
@AWTrost I installed via Preferences menu, currently on version 4.2.0 with the same problem so it's definitely an issue with the current version.

**Comment by quickgold192** [comment]
As far as reproducing it goes, I disabled all my non-core packages and just started writing a python script:
![ctagss](https://cloud.githubusercontent.com/assets/13375736/9412402/592caf52-47e0-11e5-9fd3-a03fa9eef7b5.jpg)

**Comment by quickgold192** [comment]
I just reinstalled, and my autocomplete still looks the same... But the goto/return from declaration commands work just fine.

**Comment by brbrr** [comment]
any update on this topic?

**Comment by jcostelloevanzia** [comment]
Im agreed with @AWTrost, probably it is something to do with the binary or ctags. Im having this problem on fedora, ctags installed from the fedora repo

**Comment by jcostelloevanzia** [comment]
Any news with this one?

**Comment by rubinred** [comment]
+1

**Comment by yongkangchen** [comment]
Fixed by f6d69910fe0408bafccb1b589fda8dbb330771b3
